src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -388,13 +411,16 @@ public String getCredentialsId() {
     public void setCredentialsId(@CheckForNull String credentialsId) {
         this.credentialsId = Util.fixEmpty(credentialsId);
     }
+    @DataBoundSetter
+    public void setCredentialsIdRaw(@CheckForNull String credentialsId) {
+        this.credentialsIdRaw = Util.fixEmpty(credentialsId);
+    }
 
     /**
      * Gets the repository owner.
      * @return the repository owner.
      */
     @Exported
-    @NonNull
```

Comment:
Any idea if any plugins depend on this one and call these methods? If so, starting to return null from this method and `getRepository` could cause issues. Also, seems better to add an explicit `@CheckForNull` or `@Nullable` annotation to make it clear that the results can be null.